src/components/form/HdInputPhone.vue

```diff
@@ -0,0 +1,368 @@
+<template>
+  <div class="phone-input">
+    <button
+      class="phone-input__selector"
+      aria-haspopup="listbox"
```

Comment:
You use `aria`!!! WE CARE ABOUT ACCESSIBILITY! 🎉 🎉 🎉 🎉 🎉  

YOU THE MVP!!!
![MVP](https://user-images.githubusercontent.com/14016028/143423350-6ae203a8-e012-4d33-82df-89ef03534d0a.gif)